### PR TITLE
(v7) Fix types, UMD build

### LINKED ISSUE #981: unpkg usage throwing react type is invalid error
**Describe the bug**
When I try to use react-day-picker I get the follow errors in my console
```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

**To Reproduce**
I can reproduce the error in the sandbox here -https://codesandbox.io/s/react-day-picker-base-vt99v

**Expected behavior**
Should be able to use the react-day-picker normally

### PR DESCRIPTION
**Pull request by gpbl**
I've been investigating the recent issues with types etc. There were multiple issues that were related how we build files.

This should fix #981 and #1009

### CODE REVIEW

File: types/MomentLocaleUtils.d.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
-import { LocaleUtils } from './utils';
+import { LocaleUtils } from './LocaleUtils';
 export const MomentLocaleUtils: LocaleUtils;
```
Comment (nstepien) [inline types/MomentLocaleUtils.d.ts:2]: @gpbl 
This type isn't correct, when trying to import
```ts
import utils from 'react-day-picker/moment';
```

The resulting type for `utils` is `{ MomentLocaleUtils: LocaleUtils }`.
I think the proper fix would be:

```ts
export default LocaleUtils;
```

Comment (gpbl) [reply types/MomentLocaleUtils.d.ts:2]: Hey thanks for the heads up! releasing a patch...

Comment (gpbl) [reply types/MomentLocaleUtils.d.ts:2]: Should be fixed in v7.4.7.

File: types/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
+export as namespace DayPicker;
+
 export { default } from './DayPicker';
-export * from './common';
-export * from './props';
-export * from './utils';
+export * from './Props';
```
Comment (mikaelpaj) [inline types/index.d.ts:4]: The props declaration filename is still in lowercase, this seems to break some types.

File: types/MomentLocaleUtils.d.ts (hunk 1 above)
Comment (nstepien) [reply types/MomentLocaleUtils.d.ts:2]: Thanks! It's working great.

File: types/index.d.ts (hunk 2 above)
Comment (gpbl) [reply types/index.d.ts:4]: Thanks @mikaelpaj checkout v7.4.8 – should have fixed it.

File: types/MomentLocaleUtils.d.ts (hunk 1 above)
Comment (gpbl) [reply types/MomentLocaleUtils.d.ts:2]: phew!!

File: types/index.d.ts (hunk 2 above)
Comment (mikaelpaj) [reply types/index.d.ts:4]: Fixed our issue, thanks for the quick reaction!

**Pull request merged**